database-connector/src/main/java/life/qbic/projectmanagement/persistence/repository/OpenbisConnector.java

```diff
@@ -145,10 +147,14 @@ private void createOpenbisProject(String spaceCodeString, ProjectCode projectCod
   private void deleteOpenbisProject(ProjectCode projectCode) {
     ProjectDeletionOptions deletionOptions = new ProjectDeletionOptions();
     deletionOptions.setReason("unknown reason in data-manager");
-    ProjectIdentifier projectIdentifier = new ProjectIdentifier(projectCode.value());
-    openBisClient.getV3()
-        .deleteProjects(openBisClient.getSessionToken(), List.of(projectIdentifier),
-            deletionOptions);
+    //OpenBis expects the projectspace and code during deletion
```

Comment:
why do we need to delete projects? I don't understand the change in context of the task 😅

Reply:
That's a good question. Basically what's happening if an invalid input is provided in the dialogue and the user clicks on the create button: 
1.) The ProjectCreationService creates a project in openBis and MariaDB 
2.) the AddExperimentToProjectService tries to creates an Experiment and wants to adds it to the Project. 
3.) Since not enough valid parameters are provided the Experiment creation fails. 
4.) Since we always expect a Project to have at least one active and valid experiment, the project creation has to be reverted, meaning it has to be deleted from OpenBis and MariaDb. 

This section of the PR fixes the currently not functional Projectdeletion in OpenBis